OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/BackupKeyringParcel.java

```diff
@@ -28,15 +28,12 @@
 
 public class BackupKeyringParcel implements Parcelable {
     public Uri mCanonicalizedPublicKeyringUri;
-    public Passphrase mSymmetricPassphrase;
```

Comment:
For explanation: the CryptoInputParcels are properly cached when used with the API without further work.